### PR TITLE
[fix][doc] Fixed binary protocol naming and descriptions

### PR DESCRIPTION
**Pull request by asafm**
### Modifications

* Fixed documentation of basic command structure to match the code

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

This change is a trivial rework / code cleanup without any test coverage.

### Documentation

Check the box below or label this PR directly.

Need to update docs? 

- [ ] `doc-required` 
(Your PR needs to update docs and you will update later)
  
- [ ] `doc-not-needed` 
(Please explain why)
  
- [x] `doc` 
(Your PR contains doc changes)

- [ ] `doc-complete`
(Docs have been already added)

### CODE REVIEW

File: site2/docs/develop-binary-protocol.md [hunk 1]
```diff
@@ -36,29 +36,29 @@ The Pulsar protocol allows for two types of commands:
...
-| `totalSize`   | The size of the frame, counting everything that comes after it (in bytes)               | 4               |
-| `commandSize` | The size of the protobuf-serialized command                                             | 4               |
-| `message`     | The protobuf message serialized in a raw binary format (rather than in protobuf format) |                 |
+| Component     | Description                                                               | Size (in bytes) |
+|:--------------|:--------------------------------------------------------------------------|:----------------|
+| `totalSize`   | The size of the frame, counting everything that comes after it (in bytes) | 4               |
+| `commandSize` | The size of the protobuf-serialized command                               | 4               |
+| `command`     | The protobuf serialized command                                           |                 |  
 
 ### Payload commands
```
Comment (codelipenghui) [inline site2/docs/develop-binary-protocol.md:45]: I think we'd better change here to `Message commands`

The Message command contains the `protobuf command + message metadata + user's payload`
I think it's easier to understand what is the message, what is the payload.

**Review by Anonymitaet — CHANGES_REQUESTED** [review]
The reason for the unsuccessful local preview is you modify the incorrect file. 
Please modify `developing-binary-protocol.md` rather than `develop-binary-protocol.md`, thanks.

File: site2/docs/developing-binary-protocol.md [hunk 2]
```diff
@@ -28,39 +28,39 @@ The Pulsar protocol allows for two types of commands:
 
 Simple (payload-free) commands have this basic structure:
 
-| Component     | Description                                                                             | Size (in bytes) |
-|:--------------|:----------------------------------------------------------------------------------------|:----------------|
-| `totalSize`   | The size of the frame, counting everything that comes after it (in bytes)               | 4               |
-| `commandSize` | The size of the protobuf-serialized command                                             | 4               |
-| `message`     | The protobuf message serialized in a raw binary format (rather than in protobuf format) |                 |
+| Component     | Description                                                               | Size (in bytes) |
```
Comment (asafm) [inline site2/docs/developing-binary-protocol.md:31]: image

File: site2/docs/developing-binary-protocol.md [hunk 3]
```diff
@@ -28,39 +28,39 @@ The Pulsar protocol allows for two types of commands:
...
 ## Broker entry metadata
 
 Broker entry metadata is stored alongside the message metadata as a serialized protobuf message.
 It is created by the broker when the message arrived at the broker and passed without changes to the consumer if configured.
 
-| Field              | Required or optional       | Description                                                                                                                   |
-|:-------------------|:----------------|:------------------------------------------------------------------------------------------------------------------------------|
-| `broker_timestamp` | Optional        | The timestamp when a message arrived at the broker (`id est` as the number of milliseconds since January 1st, 1970 in UTC)      |
-| `index`            | Optional        | The index of the message. It is assigned by the broker.
+| Field              | Required or optional | Description                                                                                                                |
```
Comment (asafm) [inline site2/docs/developing-binary-protocol.md:60]: image

**Comment by asafm** [comment]
@codelipenghui Changed

**Review by Anonymitaet — APPROVED** [review]

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**